de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -431,22 +429,15 @@ public void onShowAllSelected() {
     @Override
     public void onShareAnalysisSelected(ShareAnalysisSelected event) {
         List<Analysis> selected = event.getAnalysisList();
-
-        AnalysisSharingViewImpl sharingView = new AnalysisSharingViewImpl();
-        final AnalysisSharingPresenter sharingPresenter = new AnalysisSharingPresenter(analysisService,
-                                                                                 selected,
-                                                                                 sharingView,
-                                                                                 collaboratorsUtil);
-       aSharingDialogProvider.get(new AsyncCallback<AnalysisSharingDialog>() {
+        aSharingDialogProvider.get(new AsyncCallback<AnalysisSharingDialog>() {
             @Override
             public void onFailure(Throwable caught) {
-
+                ErrorHandler.post(caught);
```

Comment:
This probably isn't an issue per se, but this code can't really ever be run -- `AsyncProviderWrapper` doesn't use the `onFailure` at all. Not sure if this is one of the dialogs I switched over or one that did the same thing, but that's probably why it was initially an empty function.